LDMP/settings.py

```diff
@@ -273,7 +273,7 @@ def __init__(self, parent=None):
 
     def load_settings(self):
 
-        buffer_checked = self.settings.value("trends_earth/region_of_interest/buffer_checked", False) == 'True'
+        buffer_checked = self.settings.value("trends_earth/region_of_interest/buffer_checked", False, type=bool) == 'True'
```

Comment:
This line is flawed, you are comparing against a string but at the same time you cast the output of `settings.value` to a `bool`. This test is always false. I suggest this instead:

```suggestion
        buffer_checked = self.settings.value("trends_earth/region_of_interest/buffer_checked", False, type=bool)
```

Reply:
good catch :)... I'll add a modification to a similar code in the same PR